apis/python/src/tiledbsoma/_types.py

```diff
@@ -18,7 +20,7 @@
     NPInteger = np.integer[npt.NBitBase]
     NPFloating = np.floating[npt.NBitBase]
     NPNDArray = npt.NDArray[np.number[npt.NBitBase]]
-    PDSeries = pd.Series[Any]
+    PDSeries = pd.Series[Any]  # type: ignore[misc]
```

Comment:
could you add a note for why there's an ignore here?

Reply:
Will do. Without the ignore, mypy complains:
> apis/python/src/tiledbsoma/_types.py:23: error: Type application targets a non-generic function or class  [misc]

I have not yet figured out how to say "a Series of any type" with the current pandas-stubs.  My best guess is that it isn't possible, but instead you have to literally itemize every type that might be supported in the Series (e.g., pd.Series[int] | pd.Series[float], | ...), which seemed a bit onerous.